.babelrc

```diff
@@ -0,0 +1,16 @@
+{
+  "presets": [
+    ["@babel/env", {
+      "targets": {
+        "browsers": [
```

Comment:
Babel RC should be placed under src/.

Otherwise it will also apply to `dist`. Then whenever people import it via `require('vue-google-maps')` they will also be required to have all these babel plugins too.

Reply:
updated.